ml/src/model.py

```
@@ -0,0 +1,156 @@
+            if isinstance(m, nn.Linear):
+                nn.init.kaiming_normal_(m.weight)
+                nn.init.zeros_(m.bias)
+
+
+
+class WideAndDeep(BaseModel):
+    def __init__(
+        self,
+        wide_input_dim,
+        deep_input_dim,
+        deep_hidden_units=[128, 64],
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)